client/my-sites/themes/auto-loading-homepage-modal.jsx

```diff
@@ -176,7 +176,7 @@ class AutoLoadingHomepageModal extends Component {
 							value="keep_current_homepage"
 							checked={ 'keep_current_homepage' === this.state.homepageAction }
 							onChange={ this.handleHomepageAction }
-							label={ translate( 'Use %(themeName)s without changing my homepage content.', {
+							label={ translate( 'Use %(themeName)s without changing my homepage.', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.